plugins/fishlim/plugin_hexchat.c

```diff
@@ -138,7 +138,7 @@ static int handle_outgoing(char *word[], char *word_eol[], void *userdata) {
 static int handle_incoming(char *word[], char *word_eol[], hexchat_event_attrs *attrs, void *userdata) {
     const char *prefix;
     const char *command;
-    const char *recipient;
+    const char *channel = hexchat_get_info(ph, "channel");
```

Comment:
I don't believe this value can be relied upon as server hooks happen before a useful context is set and some events like NOTICE have no context.